docs/docs/reference/release-notes/image-migration-guide.md

```diff
@@ -0,0 +1,232 @@
+
+#### Component changes
+
+The last change is to the JSX component. The import name is potentially different, and the query result is also different.
+
+```javascript
+// import Img from "gatsby-image"
+import { GatsbyImage } from "gatsby-plugin-image"
+
+const HomePage = ({ data }) => {
+  return (
+    // <Img fixed={data.file.childImageSharp.fixed} />
+    <GatsbyImage image={data.file.childImageSharp.gatsbyImageData} />
+  )
+}
+```
+
+### API Changes
+
+In addition to the syntax changes for using `gatsby-plugin-image`, there are also changes to the API that affect the resolver arguments (and `StaticImage`).
+
+#### fluid
+
+The `fluid` image type has been deprecated in favor of two alternatives.
+
+The first is an image type called `fullWidth`. This image is designed to be used for things like hero images and generates image sizes accordingly. Instead of passing something like `maxWidth`, it takes an array called `breakpoints` that will generate images designed for those screen sizes. In most cases you will not need to provide these, as there are default values that work for most sites. Note that `fullWidth` images will expand to fit the width of their container, even if that width is larger than the source image.
+
+The second is a responsive image type called `constrained` that generates smaller images, but nothing larger than the original image source size. Additionally, you can pass `width` and/or `height` to limit the displayed image to that size. In this case, larger images may be generated for high-density screens.
+
+#### maxWidth
+
+`maxWidth` and `maxHeight` are deprecated for all image types. For `constrained` and `fixed` images, use `width` and `height`. For `fullWidth` images, look at the `breakpoints` option.
+
+#### aspectRatio
+
+`aspectRatio` is a new argument that takes a number (or fraction). If you pass it without also passing in `width` or `height`, it will default to using the source image width and then adjusting the height to the correct aspect ratio. Alternatively, you can pass your own `width` or `height` and it will set the other dimension. Passing both `height` and `width` will cause `aspectRatio` to be ignored in favor of the inferred aspect ratio.
+
+#### formats
+
+Previously, images generated their own type, e.g. JPG, PNG, etc. You could also generate WebP images when using the appropriate fragment. This is now controlled using the `formats` argument. This field takes an array, `[AUTO, WEBP]` by default, where `AUTO` means the same format as the source image.`AVIF` is now also a valid format.
+
+#### Options nested inside objects
+
+Previously, transformations like `grayscale` and quality options such as `pngQuality` were top level query arguments. This has changed.
+
+`grayscale` now exists within the `transformOptions` argument, and `pngQuality` inside `pngOptions`. See the [`gatsby-plugin-image` How-To Guide](/docs/how-to/images-and-media/using-gatsby-plugin-image/) for specifics.
```

Comment:
```suggestion
`grayscale` now exists within the `transformOptions` argument, and `pngQuality` becomes `quality` inside `pngOptions`. See the [`gatsby-plugin-image` Reference Guide](/docs/reference/built-in-components/gatsby-plugin-image) for specifics.
```

The reference guide is probably a better choice, as it documents all of the options.